src/main/java/org/walkmod/javalang/ast/stmt/CatchClause.java

```diff
@@ -47,10 +47,10 @@ public CatchClause(int exceptModifier, List<AnnotationExpr> exceptAnnotations, L
     }
 
     public CatchClause(final int beginLine, final int beginColumn, final int endLine, final int endColumn,
-            final int exceptModifier, final List<AnnotationExpr> exceptAnnotations, final List<Type> exceptTypes,
+            final int exceptModifier, final List<AnnotationExpr> exceptAnnotations, final int modifierBeginLine, final int modifierBeginColumn, final List<Type> exceptTypes,
             final VariableDeclaratorId exceptId, final BlockStmt catchBlock) {
         super(beginLine, beginColumn, endLine, endColumn);
-        setExcept(new MultiTypeParameter(beginLine, beginColumn, endLine, endColumn, exceptModifier, exceptAnnotations,
+        setExcept(new MultiTypeParameter((modifierBeginLine!=-1)?modifierBeginLine:exceptTypes.get(0).getBeginLine(), (modifierBeginColumn!=-1)?modifierBeginColumn:exceptTypes.get(0).getBeginColumn(), exceptId.getEndLine(), exceptId.getEndColumn(), exceptModifier, exceptAnnotations,
```

Comment:
Does it makes more sense in this case, just add the `MultiTypeParameter` as argument instead of exposing the internal properties of this class?

Reply:
Sorry, I didn't get your point. `MultiTypeParameter` is already an argument in this piece of code.

Reply:
sorry for the delayed response. My point is if it can be an argument of the `CatchClause` constructor, instead of setting the `except` to avoid exposing the stucture of `MultitypeParameter` in the constructor.